lib/actions/rootActions.js

```diff
@@ -2,20 +2,21 @@ import * as actions from './actionTypes';
 import { client, keenWebHost, keenWebFetchOptions } from 'Client/index';
 
 export const addDashboardItem = (title = 'Untitled') => dispatch => {
-  fetch(`https://${keenWebHost}/projects/${client.projectId()}/dashboards/`, {
-    method: 'post',
-    body: JSON.stringify({ title }),
-    ...keenWebFetchOptions
-  })
-    .then(res =>
-      res.json().then(data => {
-        dispatch({
-          type: actions.ADD_DASHBOARD_ITEM,
-          dashboardInfo: data
-        });
-      })
-    )
-    .catch(err => console.error(err));
+  keenWebHost !== 'none' &&
```

Comment:
That condition shouldn't be a part of a Redux action. I suggest to move it to container and prevent dispatching for this use case.